packages/discussions/src/utils/posts.ts

```diff
@@ -15,7 +15,13 @@ export function parseDiscussionURI(discussion: string): IDiscussionParams {
   const source = uri.protocol.slice(0, -1); // removes ":"
   const type = uri.hostname;
   const [, identifier] = uri.pathname.split("/");
-  const [id, layer = null] = identifier.split("_");
+  let id, layer;
+  if (!identifier) {
+    id = null;
+    layer = null;
+  } else {
+    [id, layer = null] = identifier.split("_");
+  }
```

Comment:
is there a reason to not use [parseDatasetId()](https://esri.github.io/hub.js/api/common/parseDatasetId/) here?